data/mods/ssb/abilities.ts

```diff
@@ -86,6 +86,24 @@ export const Abilities: {[k: string]: ModdedAbilityData} = {
 		isBreakable: true,
 	},
 
+	// Archas
+	saintlybullet: {
+		shortDesc: "Snipe Shot has 1.5x power and heals the user by 1/8 (or 1/6 on a crit) of its max HP.",
+		name: "Saintly Bullet",
+		onBasePowerPriority: 19,
+		onBasePower(basePower, attacker, defender, move) {
+			if (move.id === 'snipeshot') {
+				return this.chainModify(1.5);
+			}
+		},
```

Comment:
for reference, move.forceSTAB or something does the same trick

Reply:
I would also use forcestab over a bp modifier bc someone will add camomons to this and then Snipe Shot would be doing too much damage